src/backend/functions/orchestrator/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "orchestrator",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Orchestrator lambda for managing transcription tasks",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "dependencies": {
+    "aws-sdk": "2.369.0",
```

Comment:
do you reckon this needs to be pinned like this or can it be fixed to a major or at least minor version? also might be better not having `aws-sdk` in dependencies list to reduce package size

Reply:
I prefer to control the version of all packages precisely because it ensures a deterministic build.